src/patternfly/components/DataToolbar/data-toolbar-group-toggle-group.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-data-toolbar__group pf-m-toggle-group{{#if data-toolbar-group--modifier}} {{data-toolbar-group--modifier}}{{/if}}{{#if data-toolbar-group-toggle-group--Reveal}} pf-m-reveal{{data-toolbar-group-toggle-group--Reveal}}{{/if}}{{#if data-toolbar-toggle-group--modifier}} {{data-toolbar-toggle-group--modifier}}{{/if}}"
+  {{#if data-toolbar-toggle-group--attribute}}
```

Comment:
should you use `data-toolbar-group-toggle-group--[modifier/attribute]` as the prefix for this partial's attribute vars? You're using `data-toolbar-group-toggle-group--Reveal`. Generally the attribute matches the partial name.